### PR TITLE
Bugfix/fix eth test

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

Fix eth test after fi analytics

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-8396

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/tasty-timers-divide.md
Comment (valpinkman) [inline .changeset/tasty-timers-divide.md:1]: Changeset not needed for this

**Review by valpinkman — APPROVED** [review]
good for me but changeset not necesary

**Review by kallen-ledger — APPROVED** [review]

**Pull request merged**